README.md

```diff
@@ -1,2 +1,115 @@
-# Mbed OS Examples for the ATECC608A
+# Mbed OS Examples for the ATECC608A secure element
 
+The examples in this repository demonstrate how to use the ATECC608A secure element with Mbed OS. The `atecc608a` example demonstrates use of the ATECC608A with Mbed Crypto. Examples of how to generate a certificate using pre-provisioned ATECC608A keys and how to use ATECC608A with Mbed TLS are yet to come.
+
+Note: To see a rendered example you can import into the Arm Online Compiler, please see our [quick start](https://os.mbed.com/docs/mbed-os/latest/quick-start/online-with-the-online-compiler.html#importing-the-code).
+
+## Prerequisites
+
+* [Install Mbed CLI](https://os.mbed.com/docs/mbed-os/latest/tools/installation-and-setup.html).
+
+* [Install the arm-none-eabi-ggc toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads).
+
+
+## Importing the example
+
+From the command-line, import the example:
+
+```sh
+git clone git@github.com:ARMmbed/mbed-os-example-atecc608a.git
+cd mbed-os-example-atecc608a/atecc608a
+mbed deploy
+```
+
+## Compiling the example
```

Comment:
With `--flash --sterm`, this section is more accurately called "Compiling and running the example"